crates/ruff_linter/src/rules/flake8_trio/rules/zero_sleep_call.rs

```diff
@@ -0,0 +1,109 @@
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::Stmt;
+use ruff_python_ast::{self as ast, Expr, ExprCall, Int};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+use crate::importer::ImportRequest;
+
+/// ## What it does
+/// Checks for calls of `trio.sleep(0)` and suggests replacement with `trio.lowlevel.checkpoint()`
+///
+/// ## Why is this bad?
+/// Although equivalent to `trio.sleep(0)`, a call to `trio.lowlevel.checkpoint()` is more
```

Comment:
Interpreted from upstream plugin, couldn't find additional info in docs or PRs on the reason. All input is welcome